res/TensorFlowLiteRecipes/SegmentSum_000/test.recipe

```diff
@@ -0,0 +1,27 @@
+operand {
+  name: "ifm"
+  type: FLOAT32
+  shape { dim: 1 dim: 4 dim: 4 dim: 3 }
+}
+operand {
+name: "segment_ids"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "1" arg: "4" arg: "4" arg: "3" 
```

Comment:
This operator need to be investigated.
I tried to create such operator in TF and it failed. On the other hand TFLite 2.2 interpreter did interpret model created by this recipe (though I am not sure it did something sane).

I have two consernt about this code:
1) number of elements in segment_ids should be equal to first dimension of input
2) According TF documentation "The standard segment_* functions assert that the segment indices are sorted. "

Reply:
I tried to fill input tensor with ones, and got garbage in the output, I think tflite interpret worked just because it assumes that all checks are done in TF.